### DISCUSSION TITLE
Multiple layer's relations in Export

### DISCUSSION THREAD

**Discussion by tomtomasgreen** [django-import-export/django-import-export#1557] [Q&A]
How to export the tables with deep relations?
Class A 
id, Aname
Class B
id, A_id, Bname
Class C
id, B_id, Cname

I want to export the table like
  Cname  Bname Aname

**Comment by matthewhegarty** [comment]
Sorry we didn't respond to this.  In future, please use [one of these methods](https://django-import-export.readthedocs.io/en/latest/faq.html#what-s-the-best-way-to-communicate-a-problem-question-or-suggestion) to ask a question.
